docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -786,7 +805,7 @@ The default is `false`.
 rpc-ws-api=["ETH","NET","WEB3"]
 ```
 
-Comma-separated APIs to enable on Websockets channel.
+Comma-separated APIs to enable on WebSockets channel.
 When you use this option, the `--rpc-ws-enabled` option must also be specified.
 The available API options are: `ETH`, `NET`, `WEB3`, `CLIQUE`, `IBFT`, `DEBUG`, and `MINER`.
```

Comment:
see missing option like in --rpc-http-api

Reply:
Fixed